### PR TITLE
commit - auth0 - highScore

### PR DESCRIPTION
**Pull request by dulaj93**

### CODE REVIEW

File: ReactApp/src/App.js [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
+      uri: "wss://simplest-game-ever.hasura.app/v1/graphql",
       options: {
         reconnect: true,
         connectionParams: {
           headers: {
-            'x-hasura-admin-secret': process.env.REACT_APP_X_HASURA_ADMIN_SECRET
-          }
-        }
-      }
+            "x-hasura-admin-secret": "game", // process.env.REACT_APP_X_HASURA_ADMIN_SECRET
```
Comment (kmchmk) [inline ReactApp/src/App.js:21]: Do not hardcode the password.

File: ReactApp/src/index.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+import { Auth0Provider } from "@auth0/auth0-react";
+
+const domain = process.env.REACT_APP_AUTH0_DOMAIN;
```
Comment (kmchmk) [inline ReactApp/src/index.js:6]: You have to set these env variables in CI. Also, you may have to update the ReadMe file.

**Review by kmchmk — CHANGES_REQUESTED** [review]

File: ReactApp/src/index.js (hunk 2 above)
Comment (dulaj93) [reply ReactApp/src/index.js]: done

File: ReactApp/src/App.js (hunk 1 above)
Comment (dulaj93) [reply ReactApp/src/App.js]: done

**Review by kmchmk — APPROVED** [review]

**Pull request merged**